AK/HashTable.h

```diff
@@ -98,7 +84,7 @@ class OrderedHashTableIterator {
     void operator--() { m_bucket = m_bucket->previous; }
 
 private:
-    explicit OrderedHashTableIterator(BucketType* bucket)
+    explicit OrderedHashTableIterator(BucketType* bucket, [[maybe_unused]] BucketType*)
```

Comment:
Do you need `[[maybe_unused]]` without a parameter name?
(Maybe just left from debugging)

Reply:
I don't think so, removed it!